### PR TITLE
Fix deploy

### PR DESCRIPTION
**Pull request by metanerd**
#### Summary
Changes for the kubernetes deploy are failing the current method of deploying. This should have minimum impact to get it working again for now.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,7 +35,7 @@ ifeq (, $(shell which tilt))
 	go run ./cmd/mattermost-mattermod/main.go
```
Comment (khos2ow) [inline Makefile:35]: The extra argument is also needed here since it's been added in line 38.

**Comment by khos2ow** [comment]
Slightly related to this PR, why not rename the file to `config/config.json`, isn't `-mattermod` already implied? To be frank this is more of a discussion than a request for change on this PR.

**Comment by metanerd** [comment]
I didn't want to make huge changes right now. I agree with you though.

**Comment by khos2ow** [comment]
I agree, it shouldn't have been in this PR anyways, just wanted to start the discussion.

@agnivade any suggestions, concerns?

**Comment by agnivade** [comment]
Totally agree. We also have a ticket for changing cmd/mattermost-mattermod to cmd/mattermod.

**Pull request merged**